utils/examples/generate-examples.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import os
```

Comment:
Could we drop os and instead use 
```suggestion
from pathlib import Path
```

Makes everything easier and more readable.
We don't use `with open` and don't have to go over all the files if we use `glob`.

Reply:
@ahankinson @lpugin @musicEnfanthen please resolve this suggestion